### PR TITLE
Add absolute path link checking to munger

### LINKED ISSUE #11385: Change absolute links to Kubernetes GitHub to relative links
For example, these should be relative links. 
```sh 
grep -r "GoogleCloudPlatform/kubernetes/blob/master/docs" docs examples
docs/user-guide/limitrange/README.md:For a detailed description of the Kubernetes resource model, see [Resources](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/resources.md)
docs/user-guide/logging-demo/Makefile:# Google Cloud Logging: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/logging.md
docs/user-guide/logging-demo/Makefile:# With Elasticsearch and Kibana logging: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/logging-elasticsearch.md
docs/user-guide/update-demo/README.md:[here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl_proxy.md).
docs/user-guide/downward-api/README.md:namespace using the [downward API](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/downward_api.md).
docs/user-guide/secrets/README.md:You can learn more about secrets [Here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/secrets.md).
docs/getting-started-guides/aws.md:An up-to-date documentation page for this tool is available here: [kubectl manual](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md)
docs/getting-started-guides/aws.md:For more information, please read [kubeconfig files](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubeconfig-file.md)
docs/admin/cluster-large.md:At v1.0, Kubernetes supports clusters up to 100 nodes with 30-50 pods per node and 1-2 container per pod (as defined in the [1.0 roadmap](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/roadmap.md#reliability-and-performance)).
docs/admin/high-availability.md:the simple [Docker based single node cluster instructions](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/docker.md),
examples/cassandra/README.md:You may also note that we are setting some Cassandra parameters (```MAX_HEAP_SIZE``` and ```HEAP_NEWSIZE```) and adding information about the [namespace](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/namespaces.md).  We also tell Kubernetes that the container exposes both the ```CQL``` and ```Thrift``` API ports.  Finally, we tell the cluster manager that we need 0.5 cpu (0.5 core).
examples/guestbook/README.md:The services in a Kubernetes cluster are discoverable inside other containers [via environment variables](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#environment-variables).
examples/guestbook/README.md:An alternative is to use the [cluster's DNS service](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#dns), if it has been enabled for the cluster.  This lets all pods do name resolution of services automatically, based on the service name.
examples/guestbook/README.md:More generally, Kubernetes supports two ways of exposing a service onto an external IP address: `NodePort`s and `LoadBalancer`s , as described [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services).
examples/k8petstore/README.md:In the mean time, because the public IP will be deprecated in Kubernetes v1, we provide other 2 scripts k8petstore-loadbalancer.sh and k8petstore-nodeport.sh. As the names suggest, they rely on LoadBalancer and NodePort respectively. More details can be found [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services). 
examples/k8petstore/k8petstore-loadbalancer.sh:#This script assumes the cloud provider is able to create a load balancer. If this not the case, you may want to check out other ways to make the frontend service accessible from outside (https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services)
```

### PR DESCRIPTION
**Pull request by lavalamp**
@thockin @bgrant0607 

Fixes #11385

### CODE REVIEW

**Comment by bgrant0607** [comment]
cc @krousey, @JanetKuo, and @mikedanese in case they are interested

**Comment by bgrant0607** [comment]
So this should fix #11385

**Comment by bgrant0607** [comment]
Are we running the munger over /examples yet?

**Comment by bgrant0607** [comment]
```
examples/cassandra/README.md:You may also note that we are setting some Cassandra parameters (```MAX_HEAP_SIZE``` and ```HEAP_NEWSIZE```) and adding information about the [namespace](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/namespaces.md).  We also tell Kubernetes that the container exposes both the ```CQL``` and ```Thrift``` API ports.  Finally, we tell the cluster manager that we need 0.5 cpu (0.5 core).
examples/guestbook/README.md:The services in a Kubernetes cluster are discoverable inside other containers [via environment variables](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#environment-variables).
examples/guestbook/README.md:An alternative is to use the [cluster's DNS service](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#dns), if it has been enabled for the cluster.  This lets all pods do name resolution of services automatically, based on the service name.
examples/guestbook/README.md:More generally, Kubernetes supports two ways of exposing a service onto an external IP address: `NodePort`s and `LoadBalancer`s , as described [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services).
examples/k8petstore/README.md:In the mean time, because the public IP will be deprecated in Kubernetes v1, we provide other 2 scripts k8petstore-loadbalancer.sh and k8petstore-nodeport.sh. As the names suggest, they rely on LoadBalancer and NodePort respectively. More details can be found [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services). 
examples/k8petstore/k8petstore-loadbalancer.sh:#This script assumes the cloud provider is able to create a load balancer. If this not the case, you may want to check out other ways to make the frontend service accessible from outside (https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services)
```

**Comment by lavalamp** [comment]
It should be running over examples. I'll double check.

**Comment by lavalamp** [comment]
Ah, it didn't run over examples because it wants manual changes in docs/ first.

**Comment by lavalamp** [comment]
OK, should be good to go now.

File: docs/devel/cli-roadmap.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ certainly want the docs that go with that version.</h1>
 # Kubernetes CLI/Configuration Roadmap
 
 See also issues with the following labels:
-* [area/config-deployment](https://github.com/GoogleCloudPlatform/kubernetes/labels/area%2Fconfig-deployment)
-* [component/CLI](https://github.com/GoogleCloudPlatform/kubernetes/labels/component%2FCLI)
-* [component/client](https://github.com/GoogleCloudPlatform/kubernetes/labels/component%2Fclient)
+* [/GoogleCloudPlatform/kubernetes/labels/area/config-deployment](https://github.com/GoogleCloudPlatform/kubernetes/labels/area/config-deployment)
+* [/GoogleCloudPlatform/kubernetes/labels/component/CLI](https://github.com/GoogleCloudPlatform/kubernetes/labels/component/CLI)
+* [/GoogleCloudPlatform/kubernetes/labels/component/client](https://github.com/GoogleCloudPlatform/kubernetes/labels/component/client)
```
Comment (lavalamp) [inline docs/devel/cli-roadmap.md]: Hm... let me fix this

Comment (bgrant0607) [inline docs/devel/cli-roadmap.md]: That's unfortunate. Maybe just make the text not match the pattern?

File: docs/design/event_compression.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Instead of a single Timestamp, each event object [contains](https://github.com/G
 
 Each binary that generates events:
  * Maintains a historical record of previously generated events:
-   * Implemented with ["Least Recently Used Cache"](https://github.com/golang/groupcache/blob/master/lru/lru.go) in [```pkg/client/record/events_cache.go```](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/pkg/client/record/events_cache.go).
+   * Implemented with ["Least Recently Used Cache"](https://github.com/golang/groupcache/blob/master/lru/lru.go) in [```pkg/client/record/events_cache.go```](../../pkg/client/record/events_cache.go).
```
Comment (bgrant0607) [inline docs/design/event_compression.md]: I saw this in another PR. Code links aren't going to work on gh_pages. I think we should punt on that, though. I don't see a good solution.

File: docs/getting-started-guides/aws.md [hunk 3]
```diff
@@ -93,10 +93,10 @@ sudo cp kubernetes/platforms/darwin/amd64/kubectl /usr/local/bin/kubectl
 sudo cp kubernetes/platforms/linux/amd64/kubectl /usr/local/bin/kubectl
 ```
 
-An up-to-date documentation page for this tool is available here: [kubectl manual](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md)
+An up-to-date documentation page for this tool is available here: [kubectl manual](../../docs/user-guide/kubectl/kubectl.md)
```
Comment (bgrant0607) [inline docs/getting-started-guides/aws.md]: Is running the munger a second time going to shorten this to `../user-guide`?

File: docs/design/event_compression.md (hunk 2 above)
Comment (lavalamp) [inline docs/design/event_compression.md]: Yeah, that's something we'd have to fix in the gh_pages .html generation.

**Comment by bgrant0607** [comment]
LGTM

File: docs/getting-started-guides/aws.md (hunk 3 above)
Comment (lavalamp) [inline docs/getting-started-guides/aws.md]: Empirically, it seems stable... but that's not what I expect it to do. Not going to solve this mystery right now.

**Comment by bgrant0607** [comment]
ok to merge whenever @lavalamp says he's done with it

**Comment by lavalamp** [comment]
I am done with this now.

**Comment by bgrant0607** [comment]
Merging

File: docs/getting-started-guides/aws.md (hunk 3 above)
Comment (thockin) [inline docs/getting-started-guides/aws.md]: We should at least understand this lest it come back to bite us in the tush.
